src/main/java/uk/gov/register/thymeleaf/ThymeleafView.java

```diff
@@ -94,6 +94,10 @@ public String getScheme() {
         return requestContext.getScheme();
     }
 
+    @SuppressWarnings("unused, used by templates")
+    public Boolean getIsGovukBranded() {
+        return requestContext.getHost().endsWith("register.gov.uk");
```

Comment:
this will break if the Host header includes the port number (eg `Host: country.register.gov.uk:443`)